src/api/middleware/company.js

```diff
@@ -27,7 +27,7 @@ const verifyMaxConcurrentOffers = (owner, publishDate, publishEndDate, offerId)
 
 const verifyMaxConcurrentOffersOnCreate = (req, res, next) => {
 
-    if (req.body?.isHidden) return next();
+    if (req.body?.isHidden && JSON.parse(req.body?.isHidden)) return next();
```

Comment:
I think we should have some general sanitiser that would parse the request of the body.

Reply:
> After finding [this thread](https://stackoverflow.com/questions/10005939/how-do-i-consume-the-json-post-data-in-an-express-application/10007542#10007542), I think it is possible that problem is that `bodyParser` is deprecated. Used in:
> 
> https://github.com/NIAEFEUP/nijobs-be/blob/fb2ffcc8d462dc96dc46f38e73b70ec91df5c3bb/src/loaders/express.js#L46
> 
> It appears the solution is to replace `bodyParser` with `express`, since we are using express 4.17. [Source](https://stackoverflow.com/questions/24330014/bodyparser-is-deprecated-express-4)

Since we are no longer using `body-parser`, should I uninstall that module ?